generators/generator-base-private.js

```diff
@@ -1550,9 +1550,9 @@ module.exports = class extends Generator {
         // Prettier is clever, it uses correct rules and correct parser according to file extension.
         let prettierFilter;
         if (this.prettierJava) {
-            prettierFilter = filter(['.yo-rc.json', '{,**/}*.{md,json,ts,tsx,scss,css,yml,java}'], { restore: true });
+            prettierFilter = filter(['.yo-rc.json', '{,**/,.jhipster/**/}*.{md,json,ts,tsx,scss,css,yml,java}'], { restore: true });
         } else {
-            prettierFilter = filter(['.yo-rc.json', '{,**/}*.{md,json,ts,tsx,scss,css,yml}'], { restore: true });
+            prettierFilter = filter(['.yo-rc.json', '{,**/,.jhipster/**/}*.{md,json,ts,tsx,scss,css,yml}'], { restore: true });
```

Comment:
```javascript
        let extensionFilterForPrettier = '{,**/,.jhipster/**/}*.{md,json,ts,tsx,scss,css,yml}';
        if (this.prettierJava) {
            extensionFilterForPrettier =  '{,**/,.jhipster/**/}*.{md,json,ts,tsx,scss,css,yml,java}';
        }
        const prettierFilter = filter(['.yo-rc.json', extensionFilterForPrettier], { restore: true });
```

Reply:
I tend to avoid if-else conditionals in favor of simple ifs, makes the code easier to read and doesn't created two paths in the code